.github/workflows/test.yml

```diff
@@ -27,13 +27,10 @@ jobs:
         pycodestyle
     - name: Tests
       run: pytest
-    - name: Remove gmpy on 2.7
-      if: matrix.python-version == 2.7
-      run: pip uninstall -y gmpy2
     - name: Run coverage tests & upload results
       env:
         PYTEST_ADDOPTS: --cov mpmath
-      if: matrix.python-version == 2.7 || matrix.python-version == 3.6
+      if: matrix.python-version == 3.5 || matrix.python-version == 3.10
```

Comment:
I think it's fine to test here just one py3 version, unless there are code path (I don't think so) which are covered by different python versions.